types/hapi__content/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for @hapi/content 6.0.0
+// Project: https://github.com/hapijs/subtext#readme
+// Definitions by: Sebastian Malton <https://github.com/nokel81>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+export interface ContentType {
+  mime: string;
+  charset?: string;
+  boundary?: string;
+}
+
+export function type(header: string | undefined): ContentType;
```

Comment:
The documentation is very good to help with these types https://hapi.dev/module/content/api/?v=5.0.2